app/src/main/java/weather/ekamp/com/weatherappkotlin/model/networking/injection/HttpClientProvider.kt

```diff
@@ -0,0 +1,28 @@
+package weather.ekamp.com.weatherappkotlin.model.networking.injection
+
+import android.app.Application
+import okhttp3.Cache
+import okhttp3.OkHttpClient
+import weather.ekamp.com.weatherappkotlin.model.networking.CacheControlInterceptor
+import java.io.File
+import javax.inject.Inject
+import javax.inject.Named
+import javax.inject.Provider
+
+/**
+ * Created by anthonyfaucogney on 19/09/2017.
+ */
+class HttpClientProvider @Inject constructor(@Named("application") var application: Application) : Provider<OkHttpClient> {
+
+    companion object {
+        private val sizeOfHttpCache: Long = 10 * 1024 * 1024 // 10 MiB
```

Comment:
Fix up the naming here so we have uppercase underscore naming instead. 
`SIZE_OF_CACHE`